### PR TITLE
Some RDB improvements

### PR DESCRIPTION
**Pull request by AmbientMalice**
I've added some rom hacks/fan translations, plus tweaked a few incorrect entries.

### CODE REVIEW

**Comment by AmbientMalice** [comment]
I removed CF=1 from Indiana Jones to bring it in line with mupen64plus. The game appears to work perfectly in mupen64plus now, but PJ64 still has freezes at the end of each level.

File: Config/Project64.rdb [hunk 1]
```diff
@@ -150,6 +150,15 @@ Bad ROM?.AutoFullScreen=False
...
 // ---- END OF HEADER, START OF ROM ENTRIES --------------------------
 //================  0-9  ================
+[B2242748-FFBD61DA-C:45]
+Good Name=007 - Goldfinger (Beta)
+32bit=No
+Counter Factor=1
+Fast SP=0
+Good Name=GOLDFINGER
+Linking=0
+RDRamSize=8388608
```
Comment (Frank-74) [inline Config/Project64.rdb]: RDB has different syntax compared to .cfg.
RDRAM Size=8
Linking=Off

Comment (AmbientMalice) [reply Config/Project64.rdb]: Fixed it. Thanks.

**Comment by LegendOfDragoon** [comment]
I'm not sure if removing `HLE GFX=No` is a good idea. Sure it benefits GLideN64 users, but it does the opposite for people who use Jabo's or PJ64's Video plugin.

I could be wrong, but I think the RDB is meant to prioritize 1st party plugins.

**Comment by AmbientMalice** [comment]
HLE GFX=No was added to these games with the accurate at that time assumption that there was no way for them to be emulated in HLE. That is no longer the case, and that assumption is causing a huge usability issue for the emulator.

PJ64-Video's LLE mode is completely broken. Jabo's isn't all that much better. So what you're going to get is:

Person tries Rogue Squadron.
It's broken.
So they download GLideN64 and install it.
It's still broken.
They google the problem.
Fixing it is complicated and involved heaps of submenus and enabling Advanced Mode.
At this point, a vast majority of casual users are going to either give up or go download mupen64plus instead.

Additionally, not that it'll actually much good because it's so broken, there's nothing stopping the HLE ucode implementations being backported to PJ64-Video.

**Comment by AmbientMalice** [comment]
>”- Arcade Edition ” is not necessary.
>It is not included in the title screen, box art etc.
It's an unofficial title that distinguishes the game from Bomberman 64. I alternatively considered labeling it Bomberman 64 (2001). It's rare for two completely different N64 games to have the exact same name, and this causes some confusion.

**Comment by cxd4** [comment]
Yeah the (J) Bomberman 64 from 1997 was called Baku Bomberman or whatever.  Still I was quite confused to see Bomberman 64 (J) and think it wasn't an alternative arcade editition when in fact it was.

**Pull request merged**